ng-dev/release/publish/external-commands.ts

```diff
@@ -263,6 +262,25 @@ export abstract class ExternalCommands {
     }
   }
 
+  /**
+   * Invokes the `bazel mod deps --lockfile_mode=update` command in order
+   * to refresh `MODULE.bazel.lock` file.
+   */
+  static async invokeBazelModDepsUpdate(projectDir: string): Promise<void> {
+    const spinner = new Spinner('Updating "MODULE.bazel.lock"');
+    try {
+      await ChildProcess.spawn(getBazelBin(), ['mod', 'dep', '--lockfile_mode=update'], {
```

Comment:
I wonder if we can force a partial update of the lock file, instead of a full refresh. 

IE just `@://package.json`, I’ll try a bit more tomorrow.